src/core_plugins/elasticsearch/lib/create_client.js

```diff
@@ -0,0 +1,52 @@
+import url from 'url';
+import util from 'util';
+import elasticsearch from 'elasticsearch';
+import Bluebird from 'bluebird';
+import { readFileSync } from 'fs';
+
+const readFile = (file) => readFileSync(file, 'utf8');
+
+module.exports = function (options) {
+  options = Object.assign({ keepAlive: true, auth: true }, options);
```

Comment:
Assign this to a different variable rather than reassigning the argument.  I'd recommend:

```js
function (optionOverrides) {
  const options = Object.assign({ keepAlive: true, auth: true }, optionOverrides);
}
```